src/components/NewDraftWizard/WizardComponents/WizardStep.tsx

```diff
@@ -0,0 +1,387 @@
+import React, { useState } from "react"
+
+import ChevronRightIcon from "@mui/icons-material/ChevronRight"
+import { Typography } from "@mui/material"
+import Button from "@mui/material/Button"
+import Collapse from "@mui/material/Collapse"
+import Grid from "@mui/material/Grid"
+import List from "@mui/material/List"
+import ListItem from "@mui/material/ListItem"
+import { makeStyles } from "@mui/styles"
+import { useNavigate } from "react-router-dom"
+import { TransitionGroup } from "react-transition-group"
+
+import WizardAlert from "./WizardAlert"
+
+import { ResponseStatus } from "constants/responseStatus"
+import { ObjectStatus, ObjectSubmissionTypes, ObjectTypes } from "constants/wizardObject"
+import { resetDraftStatus } from "features/draftStatusSlice"
+import { setFocus } from "features/focusSlice"
+import { updateStatus } from "features/statusMessageSlice"
+import { resetCurrentObject, setCurrentObject } from "features/wizardCurrentObjectSlice"
+import { setObjectType, resetObjectType } from "features/wizardObjectTypeSlice"
+import { updateStep } from "features/wizardStepObjectSlice"
+import { setSubmissionType } from "features/wizardSubmissionTypeSlice"
+import { useAppDispatch, useAppSelector } from "hooks"
+import draftAPIService from "services/draftAPI"
+import objectAPIService from "services/objectAPI"
+import { ObjectInsideFolderWithTags } from "types"
+import { pathWithLocale } from "utils"
+
+const useStyles = makeStyles(theme => {
```

Comment:
Since `makeStyles/withStyles` may be deprecated in the MUI v.6, we are trying to move to `sx` and `styled` instead, so I think we could also start using `sx` and `styled` here ? This could be done in another PR as well.

Reply:
I can see that we use `makeStyles/withStyles` in multiple different components. Maybe handle all of these in a separate PR?

Reply:
I think we can also do it like that, it could be a big PR. So far I have been fixing gradually in my PRs, but I'm fine with whichever option :)

Reply:
Good point that single PR for MUI custom style migration could be a big one. Gradually would be a better approach. I'll migrate styles for this PR.

Reply:
@hannyle, latest changes introduces migrated styling methods.

Reply:
Looks good to me 👍 . These shortcuts are also useful for us: https://mui.com/system/the-sx-prop/#spacing

Reply:
Thanks! Aliases look similar to Bootstrap equivalents which are nice to have :)